src/core/scrollcomponent/web/ScrollViewer.js

```diff
@@ -41,7 +42,13 @@ export default class ScrollViewer extends React.Component {
         if (this._throttleFunction) {
             window.removeEventListener("scroll", this._throttleFunction);
             if (this.refs.mainDiv) {
-                this.refs.mainDiv.removeEventListener("scroll", this._onScroll);
+                this.refs.mainDiv.removeEventListener("scroll", this._throttleFunction);
```

Comment:
What about touch?

Reply:
Touch works out of the box, no need to handle any special cases.

Reply:
Great